modules/text_server_fb/thorvg_svg_in_ot.h

```diff
@@ -0,0 +1,86 @@
+/*************************************************************************/
+/*  thorvg_svg_in_ot.h                                                   */
+/*************************************************************************/
+/*                       This file is part of:                           */
+/*                           GODOT ENGINE                                */
+/*                      https://godotengine.org                          */
+/*************************************************************************/
+/* Copyright (c) 2007-2022 Juan Linietsky, Ariel Manzur.                 */
+/* Copyright (c) 2014-2022 Godot Engine contributors (cf. AUTHORS.md).   */
+/*                                                                       */
+/* Permission is hereby granted, free of charge, to any person obtaining */
+/* a copy of this software and associated documentation files (the       */
+/* "Software"), to deal in the Software without restriction, including   */
+/* without limitation the rights to use, copy, modify, merge, publish,   */
+/* distribute, sublicense, and/or sell copies of the Software, and to    */
+/* permit persons to whom the Software is furnished to do so, subject to */
+/* the following conditions:                                             */
+/*                                                                       */
+/* The above copyright notice and this permission notice shall be        */
+/* included in all copies or substantial portions of the Software.       */
+/*                                                                       */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,       */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF    */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.*/
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY  */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,  */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE     */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                */
+/*************************************************************************/
+
+#ifndef THORVG_SVG_IN_OT_H
```

Comment:
If this is the exact same code in both modules, maybe we could avoid the duplication by allowing ourselves a relative include of the files in the other module? I.e. have `text_server_fb` code compile and look for `../text_server_adv/thorvg_svg_in_ot.h`?

It's a bit dirty because it means that `text_server_fb` couldn't build if `text_server_adv` is deleted from the tree, but that might be better than fully duplicating the code?

Also, how is the current duplication handled if both modules are compiled together?

Reply:
> It's a bit dirty because it means that text_server_fb couldn't build if text_server_adv is deleted from the tree, but that might be better than fully duplicating the code?

For the header it's probably OK, but for the `thorvg_svg_in_ot.cpp` it might be an issue.

> Also, how is the current duplication handled if both modules are compiled together?

It's in the different compile units and is not used from any other code, so should not be an issue.